diff -ur adduser.orig/adduser.conf adduser/adduser.conf
--- adduser.orig/adduser.conf	2016-06-26 16:20:46.000000000 -0400
+++ adduser/adduser.conf	2023-02-10 12:47:06.120000000 -0500
@@ -3,7 +3,7 @@
 
 # The DSHELL variable specifies the default login shell on your
 # system.
-DSHELL=/bin/bash
+DSHELL=/bin/ksh
 
 # The DHOME variable specifies the directory containing users' home
 # directories.
diff -ur adduser.orig/examples/adduser.local.conf.examples/adduser.conf adduser/examples/adduser.local.conf.examples/adduser.conf
--- adduser.orig/examples/adduser.local.conf.examples/adduser.conf	2016-06-26 16:46:33.000000000 -0400
+++ adduser/examples/adduser.local.conf.examples/adduser.conf	2023-02-10 12:47:52.128000000 -0500
@@ -8,7 +8,7 @@
 
 # The DSHELL variable specifies the default login shell on your
 # system.
-DSHELL=/bin/bash
+DSHELL=/bin/ksh
 
 # The DHOME variable specifies the directory containing users' home
 # directories.
