--- GNUmakefile.orig	2021-01-28 14:40:52.000000000 -0500
+++ GNUmakefile	2022-09-24 15:44:30.000000000 -0400
@@ -5,7 +5,7 @@
 
 include config.mk
 
-override CFLAGS:=-I. -Ilibopenbsd -O2 -Wall -Wextra ${OS_CFLAGS} ${CFLAGS}
+override CFLAGS:=-I. -Ilibopenbsd -O2 -Wall -Wextra ${OS_CFLAGS} ${CFLAGS} ${EXTRA_CPPFLAGS} ${EXTRA_CFLAGS}
 
 all: ${PROG}
 
@@ -13,7 +13,7 @@
 OBJS:=	${OBJS:.c=.o}
 
 ${PROG}: ${OBJS}
-	${CC} ${CFLAGS} $^ -o $@ ${LDFLAGS} ${LDLIBS}
+	${CC} ${CFLAGS} $^ -o $@ ${LDFLAGS} ${EXTRA_LDFLAGS} ${LDLIBS}
 
 install: ${PROG} ${PAM_DOAS} ${MAN}
 	mkdir -p -m 0755 ${DESTDIR}${BINDIR}
