diff -urN base-files.orig/debian/rules base-files/debian/rules
--- base-files.orig/debian/rules	2019-07-09 05:00:00.000000000 -0400
+++ base-files/debian/rules	2023-02-16 14:12:04.624000000 -0500
@@ -8,7 +8,7 @@
   OSNAME=GNU/Hurd
 endif
 
-VENDORFILE = debian
+VENDORFILE = ragnarok
 DESTDIR = debian/base-files
 
 %:
diff -urN base-files.orig/etc/lsb-release base-files/etc/lsb-release
--- base-files.orig/etc/lsb-release	1969-12-31 19:00:00.000000000 -0500
+++ base-files/etc/lsb-release	2023-02-16 14:09:37.468000000 -0500
@@ -0,0 +1,4 @@
+DISTRIB_ID=Ragnarok
+DISTRIB_RELEASE=0.1
+DISTRIB_CODENAME=dev
+DISTRIB_DESCRIPTION="Ragnarok 0.1"
diff -urN base-files.orig/etc/os-release base-files/etc/os-release
--- base-files.orig/etc/os-release	2021-04-10 16:00:00.000000000 -0400
+++ base-files/etc/os-release	2023-02-16 14:09:37.468000000 -0500
@@ -1,9 +1,10 @@
-PRETTY_NAME="Debian #OSNAME# 11 (bullseye)"
-NAME="Debian #OSNAME#"
-VERSION_ID="11"
-VERSION="11 (bullseye)"
-VERSION_CODENAME=bullseye
-ID=debian
-HOME_URL="https://www.debian.org/"
-SUPPORT_URL="https://www.debian.org/support"
-BUG_REPORT_URL="https://bugs.debian.org/"
+PRETTY_NAME="Ragnarok 0.1 (dev)"
+NAME="Ragnarok"
+VERSION_ID="0.1"
+VERSION="0.1 (dev)"
+VERSION_CODENAME=dev
+ID=ragnarok
+ID_LIKE=debian
+HOME_URL="https://RagnarokOS.github.io/"
+SUPPORT_URL="https://RagnarokOS.github.io/contact.html"
+BUG_REPORT_URL="https://github.com/RagnarokOS/src/issues"
diff -urN base-files.orig/origins/ragnarok base-files/origins/ragnarok
--- base-files.orig/origins/ragnarok	1969-12-31 19:00:00.000000000 -0500
+++ base-files/origins/ragnarok	2023-02-16 14:09:37.472000000 -0500
@@ -0,0 +1,4 @@
+Vendor: Ragnarok
+Vendor-URL: https://RagnarokOS.github.io
+Bugs: https://github.com/RagnarokOS/src/issues
+Parent: Debian
diff -urN base-files.orig/share/info.dir base-files/share/info.dir
--- base-files.orig/share/info.dir	2006-04-15 12:36:50.000000000 -0400
+++ base-files/share/info.dir	2023-02-16 14:09:37.472000000 -0500
@@ -11,7 +11,7 @@
   it.
   --- PLEASE ADD DOCUMENTATION TO THIS TREE. (See INFO topic first.) ---
 
-In Debian #OSNAME#, Info `dir' entries are added with the command
+In Ragnarok, Info `dir' entries are added with the command
 `install-info'.  Please refer to install-info(8) for usage details.
 
 * Menu: The list of major topics begins on the next line.
diff -urN base-files.orig/share/motd base-files/share/motd
--- base-files.orig/share/motd	2012-06-09 06:00:00.000000000 -0400
+++ base-files/share/motd	2023-02-16 14:09:37.472000000 -0500
@@ -1,7 +1,7 @@
 
-The programs included with the Debian #OSNAME# system are free software;
+The programs included with the Ragnarok system are free software;
 the exact distribution terms for each program are described in the
 individual files in /usr/share/doc/*/copyright.
 
-Debian #OSNAME# comes with ABSOLUTELY NO WARRANTY, to the extent
-permitted by applicable law.
+Ragnarok comes with ABSOLUTELY NO WARRANTY, to the extent permitted
+by applicable law.
